Core/Core/Extensions/OperationQueueExtensions.swift

```diff
@@ -22,7 +22,7 @@ extension OperationQueue {
     public func addOperationWithErrorHandling(_ group: OperationSet, sendErrorsTo view: ErrorViewController?) {
         addOperation(group, errorHandler: { error in
             if let error = error {
-                view?.showError(error)
+                DispatchQueue.main.async { view?.showError(error) }
```

Comment:
Did you get an error with this?  ErrorViewControlller already dispatches errors on the main thread, although, it looks we don't dispatch `showError` on the main thread when it's a string.  🙁

Reply:
I got a crash here.

Reply:
The view has a showError method that stops animating a UIActivityIndicator.